worlds/ladx/ItemIconGuessing.py

```diff
@@ -0,0 +1,231 @@
+    'HERB': 'GOLD_LEAF',
+    'CARAPACE': 'SEASHELL',
+    'CONCH': 'SEASHELL',
+    'SHELL': 'SEASHELL',
+    'BOOMER': 'BOOMERANG',
+    'BEAST': 'BOWWOW',
+    'PET': 'BOWWOW',
+    'MISSILE': 'SINGLE_ARROW',
+    'QUIVER': 'SINGLE_ARROW',
+    'BIRD': 'ROOSTER',
+    'CHICKEN': 'ROOSTER',
+    'COJIRO': 'ROOSTER', # Ocarina of Time
+    'CUCCO': 'ROOSTER',
+    'FLY': 'ROOSTER',
+    'GRIFFIN': 'ROOSTER',
+    'GRYPHON': 'ROOSTER',
+    'ARMOR': 'BLUE_TUNIC',
+    'DRAGONARMOR': 'BLUE_TUNIC', # Final Fantasy 1
+    'MAIL': 'BLUE_TUNIC',
+    'SUIT': 'BLUE_TUNIC',
+    'TANK': 'HEART_CONTAINER',
+    'FUNGAL': 'TOADSTOOL',
+    'FUNGUS': 'TOADSTOOL',
+    'MUSHROOM': 'TOADSTOOL',
+    'SHROOM': 'TOADSTOOL',
+    'NUT': 'GUARDIAN_ACORN',
+    'SEED': 'GUARDIAN_ACORN',
+    'KEY': 'KEY', # Without this, keys show up as nightmare keys
+    'FOSSIL': 'STONE_BEAK',
+    'RELIC': 'STONE_BEAK',
+    'BOLERO': 'SONG1',
+    'LULLABY': 'SONG1',
+    'MELODY': 'SONG1',
+    'MINUET': 'SONG1',
+    'NOCTURNE': 'SONG1',
+    'PRELUDE': 'SONG1',
+    'REQUIEM': 'SONG1',
+    'SERENADE': 'SONG1',
+    'SONG': 'SONG1',
+    'FISH': 'SONG2',
+    'SURF': 'SONG2',
+    'FROG': 'SONG3',
+    'CELLO': 'INSTRUMENT1',
+    'GUITAR': 'INSTRUMENT1',
+    'LUTE': 'INSTRUMENT1',
+    'VIOLIN': 'INSTRUMENT1',
+    'HORN': 'INSTRUMENT2',
+    'BELL': 'INSTRUMENT3',
+    'CHIME': 'INSTRUMENT3',
+    'HARP': 'INSTRUMENT4',
+    'KANTELE': 'INSTRUMENT4',
+    'MARIMBA': 'INSTRUMENT5',
+    'XYLOPHONE': 'INSTRUMENT5',
+    'TRIANGLE': 'INSTRUMENT6',
+    'KEYBOARD': 'INSTRUMENT7',
+    'ORGAN': 'INSTRUMENT7',
+    'PIANO': 'INSTRUMENT7',
+    'DRUM': 'INSTRUMENT8',
+    'DINOSAUR': 'TRADING_ITEM_YOSHI_DOLL',
+    'DRAGON': 'TRADING_ITEM_YOSHI_DOLL',
+    'TOY': 'TRADING_ITEM_YOSHI_DOLL',
+    'HAIRBAND': 'TRADING_ITEM_RIBBON',
+    'HAIRPIN': 'TRADING_ITEM_RIBBON',
+    'CAN': 'TRADING_ITEM_DOG_FOOD',
+    'BANANA': 'TRADING_ITEM_BANANAS',
+    'BRANCH': 'TRADING_ITEM_STICK',
+    'TWIG': 'TRADING_ITEM_STICK',
+    'BEEHIVE': 'TRADING_ITEM_HONEYCOMB',
+    'HIVE': 'TRADING_ITEM_HONEYCOMB',
+    'HONEY': 'TRADING_ITEM_HONEYCOMB',
+    'FRUIT': 'TRADING_ITEM_PINEAPPLE',
+    'GOURD': 'TRADING_ITEM_PINEAPPLE',
+    'FLOWER': 'TRADING_ITEM_HIBISCUS',
+    'PETAL': 'TRADING_ITEM_HIBISCUS',
+    'CARD': 'TRADING_ITEM_LETTER',
+    'MESSAGE': 'TRADING_ITEM_LETTER',
+    'DUSTY': 'TRADING_ITEM_BROOM',  # Tunic: a broom
+    'SWEEP': 'TRADING_ITEM_BROOM',
+    'CLAW': 'TRADING_ITEM_FISHING_HOOK',
+    'AMULET': 'TRADING_ITEM_NECKLACE',
+    'BEADS': 'TRADING_ITEM_NECKLACE',
+    'PEARLS': 'TRADING_ITEM_NECKLACE',
+    'PENDANT': 'TRADING_ITEM_NECKLACE',
+    'ROSARY': 'TRADING_ITEM_NECKLACE',
+    'FINDER': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'LENS': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'MIRROR': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'SCOPE': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'CHAOS': 'PIECE_OF_POWER', # Chaos emerald
+    'TRIANGLE': 'PIECE_OF_POWER',
```

Comment:
```suggestion
    'TRIANGLES': 'PIECE_OF_POWER',
```
Since currently there's a duplicate key with the triangle instrument

Reply:
Oh true, i think other cases of "triangle" would probably match piece of power better than the instrument though, so I think I'll just remove the instrument line.

Reply:
Can we assume all items can be used as values here? If not, maybe keep that instrument in commented out, with a note that it's unused, so in the future if someone wants to add more they can see that option there

Reply:
It is all items but I'll reorganize it for better clarity.